openshift/_python36.dc.json

```diff
@@ -58,6 +58,11 @@
       "required": false,
       "value": "/opt/app-root/src/document_uploads"
     },
+    {
+      "name": "MDS_NRIS",
```

Comment:
Can you make the Name more descriptive? Like `MDS_NRIS_SECRET_NAME` or something?

Reply:
I suggest MDS_NRIS_API_USERNAME and MDS_NRIS_API_PASSWORD to follow the WSO2/OAuth2 terms.

Reply:
I suggest MDS_NRIS_API_USERNAME and MDS_NRIS_API_PASSWORD to follow the WSO2/OAuth2 terms.

Reply:
Changed to MDS_NRIS_API_SECRET